wordpress-mobile/WordPress-FluxC-Android
# Revisions DB migration script

Forgot to add a DB migration script, which might cause some issue when upgrading/downgrading app versions.

To test:

1. Install the current `develop` branch of wpandroid with flux-c reference pointed to this PR `74b85fe6a290278d35952394d4e749ae5962d3e6`
2. Revert the flux-c reference and install the app again.
3. Make sure app installs correctly.
